e2e/testcases/stress_test.go

```diff
@@ -342,8 +341,7 @@ func TestStressManyDeployments(t *testing.T) {
 
 	// Validate that the resources sync without the reconciler running out of
 	// memory, getting OOMKilled, and crash looping.
-	// Skip ResourceGroup checks because it takes too long for this many Deployments to reconcile
-	nt.Must(nt.WatchForAllSyncs(nomostest.SkipAllResourceGroupChecks()))
+	nt.Must(nt.WatchForAllSyncs())
```

Comment:
Why is this expected to reconcile now?

Reply:
It should always have reconciled. The reconcile timeout is set to 5m in the applier, which makes each apply attempt wait the normal amount of time, WatchObject uses the DefaultWaitTimeout of 30m for stress tests, and the WatchObject for the RG starts after the WatchObject for the RSync succeeds. So it has plenty of time for the Deployment Pods to schedule, even on autopilot.

Reply:
I have doubts about that, but I suppose we'll see. I don't see changes here that address the existing failures for this test

Reply:
I'm running it locally. Will know shortly.

Reply:
Looks like you're right. Only about 350 are being correctly scheduled on the standard stress clusters.

```
status:
  conditions:
  - lastProbeTime: null
    lastTransitionTime: "2025-03-12T20:53:42Z"
    message: '0/3 nodes are available: 3 Too many pods. preemption: 0/3 nodes are
      available: 3 No preemption victims found for incoming pod.'
    reason: Unschedulable
    status: "False"
    type: PodScheduled
  phase: Pending
  qosClass: Guaranteed
```

Not super clear why the nodes are "not available" tho.

Each of the nodes has a lot of available Memory and CPU too.

The ReplicaSets also seem to be timing out reporting status.

Reply:
Ah. It's probably because of the node's IP limit, which is 110 by default.

Reply:
Yup. I got it working with 4 nodes per zone in 3 zones.

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1601